### DISCUSSION TITLE
"Failed to sync Longhorn setting longhorn-system/v2-data-engine" messages every hour

### DISCUSSION THREAD

**Discussion by eoprede** [longhorn/longhorn#6444] [General]
Hello,
I'm running Longhorn 1.5.1 and I have noticed after upgrade that I am getting following messages every hour:
```
time="2023-08-03T01:29:17Z" level=error msg="Failed to sync Longhorn setting longhorn-system/v2-data-engine" controller=longhorn-setting error="failed to sync setting for longhorn-system/v2-data-engine: cannot apply v2-data-engine setting to Longhorn workloads when there are attached volumes" node=slave2
```
```
time="2023-08-03T01:29:17Z" level=error msg="Dropping Longhorn setting longhorn-system/v2-data-engine out of the queue" controller=longhorn-setting error="failed to sync setting for longhorn-system/v2-data-engine: cannot apply v2-data-engine setting to Longhorn workloads when there are attached volumes" node=slave2
```
I have similar messages for `longhorn-system/storage-network` as well.
I have total of 3 nodes, only 1 of them gets these messages. As far as I know, nothing on that node is configured differently. 
Could you point me to what could be the reason for these errors?
Thanks!

**Comment by c3y1huang** [comment]
You can disregard those misleading errors. They are from the [periodic sync with the setting controller](https://github.com/longhorn/longhorn-manager/blob/v1.5.x/controller/setting_controller.go#L51).

**Reply by c3y1huang** [reply]
https://github.com/longhorn/longhorn/issues/6445
